appium/webdriver/extensions/performance.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium import webdriver
+from ..mobilecommand import MobileCommand as Command
+
+
+class Performance(webdriver.Remote):
+
+    def get_performance_data(self, package_name, data_type, data_read_timeout=None):
+        """Returns the information of the system state
+        which is supported to read as like cpu, memory, network traffic, and battery
```

Comment:
Could you address about "For Android"? Because Python client has all of APIs in one module